### PR TITLE
[Testing] Add link request method & add type hint

### PR DESCRIPTION
**Pull request by mohamedGasmii**

### CODE REVIEW

File: testing.rst [hunk 1]
```diff
@@ -513,14 +513,14 @@ returns a ``Crawler`` instance.
...
-        $uri,
+        string $method,
+        string $uri,
         array $parameters = [],
         array $files = [],
         array $server = [],
-        $content = null,
-        $changeHistory = true
-    )
+        string $content = null,
```
Comment (OskarStark) [inline testing.rst:521]:
```suggestion
        ?string $content = null,
```
?

Comment (mohamedGasmii) [reply testing.rst:521]: It makes more sense with '?' cause it's nullable but i've followed the exact declaration : 
**public function request(string $method, string $uri, array $parameters = [], array $files = [], array $server = [], string $content = null, bool $changeHistory = true)**

Comment (javiereguiluz) [reply testing.rst:521]: I agree with Mohamed. Thanks!

**Comment by javiereguiluz** [comment]
Thank you @mohamedGasmii.

**Pull request merged**